extensions/vault/model/src/main/java/io/quarkus/vault/runtime/client/dto/pki/VaultPKIEnableBody.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vault.runtime.client.dto.pki;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class VaultPKIEnableBody {
+
+    public static class Config {
+
+        @JsonProperty("default_lease_ttl")
+        public String defaultLeaseTimeToLive;
+
+        @JsonProperty("max_lease_ttl")
+        public String maxLeaseTimeToLive;
+
+    }
+
+    public String type = "pki";
```

Comment:
There is no default for 'type` when calling the API. That's the engine type you want to mount. It's here because this is always enabling the `PKI` engine.

Reply:
Removed when moved to internal system backend bean.